src/app/containers/Transactions/index.js

```diff
@@ -61,7 +62,14 @@ export default class Transactions extends Component {
     )
     return (
       // <ul style="overflow: hidden;">{listItems}</ul>
-      <textarea readOnly style='border: 0; bottom: 0;' rows='20' cols='40' name='transactionList'>{listItems}</textarea>
+      <textarea
+        readOnly
+        style={ { border: 0, bottom: 0 } }
```

Comment:
Should this be handled in some sort of reset or moved to CSS?

Reply:
Not sure. I was just fixing this part so it worked. I think transactions needs a good once over in general.